### ISSUE TITLE
Proposal:  Merge to spec more frequently

### ISSUE THREAD

**Issue by mmatsa** [graphql/graphql-over-http#148]
@abernix @balshor @benjie @deinok @ErikWittern @jaydenseric @michaelstaib @mike-marcacci @mmatsa @spawnia @sungam3r @enisdenjo @IvanGoncharov

Discussion with @IvanGoncharov @ghmcadams @mmatsa 

Issue/Background:  We want to move along discussion
- The spec isn't evolving quickly enough - it's hard to get reviews of PRs and consensus
- The spec isn't moving quickly enough - we want to release it while it's still relevant
- With more PRs merging we might attract more people to participate more actively
- The spec is short
- It's not a spec yet, no one implements it yet, it is breaking on all servers, so it's not critical to keep `master` branch always clean, and it's ok to back out changes

Summary:
- **We're trying to separate "Preliminary" stage from "Draft" stage to get the spec moving more quickly until we get to Draft stage.**

Proposal:
- We already emphasize that the spec is a draft and might change dramatically.  Make that stronger.
- If a PR has no requests for changes for 2 weeks then it should be merged by one of the maintainers
- If anyone has an objection later, they just open a PR to make the change and it goes through the same process
- Optional: When there is lots of consensus but not 100% full consensus then:
   - We might merge the consensus-view and debate modifying it in parallel
   - Anyone can extract the non-controversial part and make a separate PR
- Spec is now in "Preliminary" stage.  
- At some point we would promote it to "Draft" stage.
   - Before release of the spec, in "Draft" stage, we have to review the spec and review all open PRs
   - At this point every merge to `master` would need strong consensus
   - Only changes that address concerns
   - Implementers could start trying things

We'd like feedback from the group.
Any concerns with the Summary?  Or the details?
Based on discussion here, we can open a PR.

**Comment by sjparsons** [comment]
@mmatsa I think this makes sense. I think it's critical to increase the pace of development of the spec and get towards a true Draft as you've defined it. Who's up for opening a PR documenting the work flow described in the Proposal?

**Comment by mmatsa** [comment]
Thanks @sjparsons for the comment.  I've been busy with other things for the last month or so, but we seem to have consensus now:
- thumbs up from @jaydenseric @benjie @enisdenjo @IvanGoncharov @sjparsons 
- no requests at all for anything different, no questions
- 27 days

I agree that's enough consensus.  I can try to write up the PR within the next week.

**Comment by mmatsa** [comment]
Approvals on the issue: @jaydenseric @benjie @enisdenjo @IvanGoncharov @sjparsons @michaelstaib @spawnia 
Approvals on the PR: @spawnia @benjie 
More than 14 days wait on the PR, no objections.
Merged.
I'm closing this issue.  If there are any modifications to make please open a new issue/PR.
